jekyll/_cci2/configuration-reference.adoc

```diff
@@ -748,6 +753,17 @@ Each job consists of the job's name as a key and a map as a value. A name should
 
 '''
 
+[#job-type]
+==== `type`
+
+Configure a job type. Options are `release`, `approval`, `build` (default). If a type is not specified, the job defaults to a `build` type.
+
+Jobs with the `release` type are used to xref:/release/configure-your-kubernetes-components#link-release[connect your pipeline configuration] to a release in the CircleCI releases UI. For full details, see the xref:/release/releases-overview#[Releases overview] page.
+
+The `approval` type is used to configure a manual approval step. No `job` configuration is required or allowed for an `approval` type job. The `approval` type is most commonly configured within a workflow. See <<type,type under workflows section>> for full details.
```

Comment:
```suggestion
The `approval` type is used to configure a manual approval step. No `job` configuration is required or allowed for an `approval` type job. The `approval` type is most commonly configured within a workflow rather than under the top-level `jobs` key. Only `approval` type jobs can have their `type` configured under `workflows`. See <<type,type under workflows section>> for full details.
```